x-pack/test/api_integration/apis/ml/job_audit_messages/clear_messages.ts

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { omit } from 'lodash';
+import { FtrProviderContext } from '../../../ftr_provider_context';
+import { getJobConfig } from './index';
+import { USER } from '../../../../functional/services/ml/security_common';
+import { COMMON_REQUEST_HEADERS } from '../../../../functional/services/ml/common_api';
+
+export default ({ getService }: FtrProviderContext) => {
+  const esArchiver = getService('esArchiver');
+  const supertest = getService('supertestWithoutAuth');
+  const ml = getService('ml');
+
+  describe('clear_messages', function () {
+    before(async () => {
+      await esArchiver.loadIfNeeded('x-pack/test/functional/es_archives/ml/farequote');
+      await ml.testResources.setKibanaTimeZoneToUTC();
+
+      for (const jobConfig of getJobConfig(1)) {
+        await ml.api.createAnomalyDetectionJob(jobConfig);
+      }
+    });
+
+    after(async () => {
+      await ml.api.cleanMlIndices();
+    });
+
+    it('should mark audit messages as cleared for provided job', async () => {
+      const timestamp = Date.now();
+
+      const { body } = await supertest
+        .put(`/api/ml/job_audit_messages/clear_messages`)
+        .auth(USER.ML_POWERUSER, ml.securityCommon.getPasswordForUser(USER.ML_POWERUSER))
+        .set(COMMON_REQUEST_HEADERS)
+        .send({
+          jobId: 'test_get_job_audit_messages_1',
+          notificationIndices: ['.ml-notifications-000002'],
```

Comment:
I'm a bit concerned about using this system index directly - in future versions this might change to `...03`, `..04`, etc. I'm not perfectly sure, but I thought there was an alias for it that we could use?

Reply:
Don't think we have 🤔 `notificationIndices` param for this API was introduces in  https://github.com/elastic/kibana/pull/106879

Reply:
Indeed it seems we don't have an alias here. But maybe we can also try to get rid of the hard coded notifications index name (similar to the PR you posted)?

Reply:
Added in https://github.com/elastic/kibana/pull/110793/commits/c12d7ff47163e2981169641dd19ecc3f777377c4